index.html

```diff
@@ -0,0 +1,46 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="style.css" />
+    <link
+      href="https://fonts.googleapis.com/css?family=Poppins"
+      rel="stylesheet"
+    />
+    <title>Document</title>
+  </head>
+  <body>
+    <header>
+      <ul>
+        <li><img src="My Logo.svg" alt="Logo svg" /></li>
+        <li><img src="Union.png" alt="Menu Button" /></li>
+      </ul>
+    </header>
+    <div class="headline">
+      <div class="headline-intro">
+        <h1>
+          I’m Mariane <br />
+          Glad to see you!
+        </h1>
+        <p>
+          I’m a software developer! I can help you build a product , feature or
+          website Look through some of my work and experience! If you like what
+          you see and have a project you need coded, don’t hesitate to contact
+          me.
+        </p>
+      </div>
+      <h3>LET’S CONNECT</h3>
+      <div class="icons">
+        <ul>
+          <li><img src="Twitter.svg" alt="Twitter svg" /></li>
+          <li><img src="Linkedin.svg" alt="Linkedin svg" /></li>
+          <li><img src="M.svg" alt="M svg" /></li>
+          <li><img src="Cat.svg" alt="Cat svg" /></li>
+          <li><img src="Hand.svg" alt="Hand svg" /></li>
+        </ul>
```

Comment:
- [ ] Great job using `<ul>` and `<li>` tags 💯. However, I suggest wrap the social links with `<a>` tag. because the social icons should be clickable 👌

> For Example 
```
<li>
   <a href="https://twitter.com"
      <img src="Twitter.svg" alt="Twitter svg" />
   </a>
</li>

....
```